coverage_point_calculator/src/lib.rs

```diff
@@ -165,12 +167,13 @@ impl CoveragePoints {
         let hex_coverage_points = hexes::calculated_coverage_points(&covered_hexes);
 
         let speedtests = speedtest::clean_speedtests(speedtests);
-        let speedtest_multiplier = speedtest::multiplier(&speedtests);
+        let (speedtest_multiplier, speedtest_avg) = speedtest::multiplier(&speedtests);
```

Comment:
rather than having a function named `multiplier` now return both the multiplier and the average when there is an existing average function, would it be better to just call the average function here first, then change the multiplier function to access that average as a param?

Reply:
I was thinking that way. 

> then change the multiplier function to access that average as a param?

I'm not sure that it will give us any benefits because `speedtest_avg` has the same type as `speedtest` (pub struct Speedtest).  I don't see how to make this function non-confusing. (Except making new type `SpeedTestAvg` that `multiplier` will accept) 

Also, there is an important check there and I'm not sure that it is worth moving it somewhere. 

```rust
if speedtests.len() < MIN_REQUIRED_SPEEDTEST_SAMPLES {
        return (dec!(0), avg);
}
```

So it looks safer (but maybe not cleaner) for me, save it as is.

Reply:
yeah its all a lil convoluted in there with some of these functions and maybe beyond scope of this PR to address